src/loader-components/ExampleTemplate.module.scss

```diff
@@ -0,0 +1,81 @@
+:local {
+  .template {
+    border: 1px solid #e8e8e8;
+    border-radius: 3px;
+    margin-bottom: 0.9375rem;
+    overflow: hidden;
+  }
+
+  .header {
+    background-color: var(--terra-dev-site-header-background-color);
+    border-bottom: 1px solid #e8e8e8;
+    padding: 0.625rem;
+  }
+
+  .content {
+    background-color: var(--terra-dev-site-template-content-background-color);
```

Comment:
should all fo these css vars be prefixed with `--terra-dev-site-example-template`

Reply:
@StephenEsser How should I name these things?

Reply:
Very carefully.

Reply:
😂

Reply:
Don't encourage him.